test/widgets/compose_box_test.dart

```diff
@@ -48,6 +48,8 @@ void main() {
     if (narrow case ChannelNarrow(:var streamId) || TopicNarrow(: var streamId)) {
       assert(store.streams.values.any((stream) => stream.streamId == streamId),
         'Add a channel with "streamId" the same as of $narrow.streamId to the store.');
+      assert(store.users.values.any((user) => user.userId == account.userId),
+        'Add an user with "userId" the same as of $account.userId to the store.');
```

Comment:
nit: "a user"

Reply:
The assertion will be removed as `account` is replaced with `zulipFeatureLevel`.